src/stream/src/executor/managed_state/join/mod.rs

```diff
@@ -190,7 +190,7 @@ impl<K: HashKey, S: StateStore> JoinHashMap<K, S> {
 
     /// Returns a mutable reference to the value of the key in the memory, if does not exist, look
     /// up in remote storage and return, if still not exist, return None.
-    pub async fn get_mut<'a>(&'a mut self, key: &K) -> Option<&'a mut HashValueType<S>> {
+    pub async fn get_mut<'a, 'b>(&'a mut self, key: &'b K) -> Option<&'a mut HashValueType<S>> {
```

Comment:
The error here is really strange. 🤣 Shall we bump the toolchain? cc @skyzh

Reply:
Then let's hold this PR for a second

Reply:
Let's bump and see if this will still happen 🤣 Have you found some related issue in rust-lang/rust?